Exercise-HTML-CSS-1/index.html

```diff
@@ -0,0 +1,145 @@
+<!DOCTYPE html>
+	<head>
+        <meta charset="utf-8">
+        <meta http-equiv="X-UA-Compatible" content="IE=edge">
+        <meta name="viewport" content="width=device-width, initial-scale=1">
+        
+		<title>GirlDevelopIt</title>
+		<link href = "css/styles.css" rel = "stylesheet" type ="text/css">
+        
+            <!-- Bootstrap -->
+        <link rel="stylesheet" href="Bootstrap/css/bootstrap.min.css" />
+        <link rel="stylesheet" href="Bootstrap/css/bootstrap.css" />
+        <!-- jQuery (necessary for Bootstrap's JavaScript plugins) -->
+        <script src="https://ajax.googleapis.com/ajax/libs/jquery/1.11.3/jquery.min.js"></script>
+        <!-- Include all compiled plugins (below), or include individual files as needed -->
+        <script src="Bootstrap/js/bootstrap.min.js"></script>
+	</head
+	<body>
+        
+		<div>
+			<img src="https://stackoverflow.blog/wp-content/uploads/2017/08/wordmark-tagline-pink-2.png">
+		</div>
+
+		<!--div about-->
+		<div>
+			<h1>About</h1>
+            
+            <div class = "row">
```

Comment:
Don't leave extra spaces in attributes. That should be written like: `<div class="row">`
Obviously you can use spaces inside the quotes, but outside the quotes you should not leave spaces. If you ever need a two-word attribute for an html tag, you should use a tag and always lower case (Ex. `<div data-content="This is content"></div>`)